src/hooks/useMediaDuration/useMediaDuration.test.tsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable react/no-multi-comp */
+import { act, fireEvent, render } from '@testing-library/react';
+import { type ReactElement, useRef } from 'react';
+import { useMediaDuration } from './useMediaDuration.js';
+
+describe('useMediaDuration', () => {
+  it('should return media duration once the video is loaded', async () => {
```

Comment:
You can now use the playwright testing setup to test this in a real browser.

Reply:
Our idea was to use `renderHook` to make the test even simpler.

But useful to know that we now could use playwright indeed :)

Reply:
The `jest-dom` doesn't implement the media events afaik so if you use the testing library to test this stuff you'd be testing your own implementation. It's worth test to check if the hook handles events correctly but a test in playwright will be more robust (and it'll make the test with `renderHook` redundant).

Reply:
We don't care about media events. We only care about 2 scenarios:
- duration exists, and no event fires
- duration does not exist yet, and event fires

how those durations are set, and the events are fired, doesn't really matter, so making the test as simple as possible with mocks makes it run way faster, and more deterministic.

Reply:
implemented